### DISCUSSION TITLE
Device mappings in watch designer

### DISCUSSION THREAD

**Discussion by kartun83** [zepp-health/zeppos-forum#251] [Developer Feedback]
Hello.
In [Watchface designer](https://watchface.zepp.com/) exported package is the same for GTR4 and GTR4 Limited edition. In app.json it's the same target.
But in zeus-cli it requires separate targets and for GTR4 LE it requires device dims 480x480.
Which one is correct ?

**Comment by zdunex25** [comment]
GTR 4: 7930112;
GTR 4W: 7930113;
GTR 4LE: 7864577;
Screen size: 466x466

**Reply by kartun83** [reply]
May be I don't get the idea.
---
`"gtr4": {
            "module": {
                "watchface": {
                    "path": "watchface/default-target/index",
                    "main": 1,
                    "editable": 0,
                    "lockscreen": 1,
                    "hightCost": 0
                }
            },
            "platforms": [
                {
                    "name": "Amazfit GTR 4",
                    "deviceSource": 7930112
                },
                {
                    "name": "Amazfit GTR 4 W",
                    "deviceSource": 7930113
                },
                {
                    "name": "Amazfit GTR 4 Limited Edition",
                    "deviceSource": 7864576
                },
                {
                    "name": "Amazfit GTR 4 Limited Edition",
                    "deviceSource": 7864577
                }
            ],
            "designWidth": 466
        },`

in terminal:
`zeus dev
warning  Unsupported targets(gtr4, gtr3_trex) have been removed.
error  Please set at least one package.
`

but this code works without errors:
`
"gtr4": {
            "module": {
                "watchface": {
                    "path": "watchface/default-target/index",
                    "main": 1,
                    "editable": 0,
                    "lockscreen": 1,
                    "hightCost": 0
                }
            },
            "platforms": [
                {
                    "name": "Amazfit GTR 4",
                    "deviceSource": 7930112
                },
                {
                    "name": "Amazfit GTR 4 W",
                    "deviceSource": 7930113
                }
            ],
            "designWidth": 466
        },
        "gtr4_le":{
            "module": {
                "watchface": {
                    "path": "watchface/default-target/index",
                    "main": 1,
                    "editable": 0,
                    "lockscreen": 1,
                    "hightCost": 0
                }
            },
            "platforms": [
                {
                    "name": "Amazfit GTR 4 Limited Edition",
                    "deviceSource": 7864576
                },
                {
                    "name": "Amazfit GTR 4 Limited Edition2",
                    "deviceSource": 7864577
                }
            ],
            "designWidth": 466
        },
`

if design width is the same for both targets, why it should be separated in app.json ?

**Comment by sunnyBob** [comment]
Thanks for your feedback, This problem has been fixed, you can run `rm -rf ~/.zepp/.zeus_device*` to clear supported devices cache and try again. @kartun83
